parlai/tasks/taskmaster2/agents.py

```diff
@@ -171,6 +171,9 @@ def custom_evaluation(
                     f'{domain}_slot_r',
                     AverageMetric(correct, len(teacher_action['slots'])),
                 )
+                self.metrics.add(
+                    'jga', AverageMetric(correct == len(teacher_action['slots']))
```

Comment:
It might be more readable for us to just compare the dictionaries (see FB:multiwoz code), but this works.